### DISCUSSION TITLE
Mass edit URLs

### DISCUSSION THREAD

**Discussion by glottisfaun0000** [FreshRSS/FreshRSS#6619] [Q&A]
Wondering if there's an internal/db option for mass editing urls. I had to replace the domain on a bunch of subscribed feeds, and the best solution I've figured out is to export OPML of the category, edit the opml, empty the category, and import the edited version. But I'm wondering if there's a programatic way to do this internally by manipulating the db or something, or if anyone has developed an extension to do so.

Thanks

**Comment by Alkarex** [comment]
I am not aware of any existing extension to do so, but an SQL update request would probably be the easiest
